docs/docs/dev/api.md

```diff
@@ -25,7 +25,7 @@ span.pre {
 This page describes the REST API that the system's frontend uses to communicate with the backend.
 
 :::{note}
-All requests should also pass an authorization header with the Bearer token returned by the  [login endpoint](#login).
+If in the configuration file `login_required` is set to true, all requests should also pass an authorization header with the Bearer token returned by the  [login endpoint](#login).
```

Comment:
@martinscooper thanks for editing the documentation to clarify that when the Bearer token is needed! When mentioning the configuration file, can we cross-reference the corresponding documentation page?

```suggestion
If in the [configuration file](configuration.md) `login_required` is set to true, all requests should also pass an authorization header with the Bearer token returned by the  [login endpoint](#login).
```